app/frontend/src/toast-ui-viewer.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "@toast-ui/editor/dist/toastui-editor-viewer" {
+  import { Viewer } from "@toast-ui/editor/types/editor";
+
+  export default Viewer;
+}
```

Comment:
Does that mean they have the right types already, but somehow not included it for the viewer?